foreman/data_refinery_foreman/surveyor/test_array_express.py

```diff
@@ -264,25 +267,27 @@ def test_experiment_object(self, mock_get):
     @patch('data_refinery_foreman.surveyor.message_queue.app.send_task')
     def test_survey(self, mock_send_task, mock_get):
         """survey generates one Batch per sample with all possible fields populated.
```

Comment:
In that case I would put it in quotes, or specifically say something like "The survey function" or "The survey() function", since it is kind of unclear that it is talking about the survey function there. Or maybe something like `ArrayExpressSurveryor.survey() generates...`?